### ISSUE TITLE
confluent-community-1.0  reported as proprietary-license

### ISSUE THREAD

**Issue by DennisClark** [nexB/scancode-toolkit#1861]
A recent scan of ksql-5.4.0-rc191218083414.tar.gz from scancode.io returned multiple hits for the very generic `proprietary-license` but none for the `confluent-community-1.0` license, which is a source-available license with a very specific text that should be recognized by scancode-toolkit. Scan results attached.

See https://www.confluent.io/confluent-community-license/ 

See https://github.com/confluentinc/ksql/releases

[ksql-5.4.0-rc191218083414.tar.gz_scan.json.zip](https://github.com/nexB/scancode-toolkit/files/3980198/ksql-5.4.0-rc191218083414.tar.gz_scan.json.zip)

**Comment by pombredanne** [comment]
@DennisClark good catch ... based on https://github.com/nexB/scancode-toolkit/search?q=Confluent&unscoped_q=Confluent there is indeed a mix of rules pointing to multiple license keys for this one.

**Comment by DennisClark** [comment]
No longer important since there does not appear to be a current community version ksql available. closing.
